<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>771</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:2, y2:0, stop:0 rgba(70,0,79), stop:1 rgba(37,8,164,1));
border-radius:20px;
}</string>
    </property>
    <widget class="QLabel" name="welcomHeader_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>821</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>ND LOGOS</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
border-radius:20px;
</string>
     </property>
     <property name="text">
      <string> Password Generator</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="backgrounWhite_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>961</width>
       <height>621</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="img_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>241</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>secure.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="lower_chBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#QCheckBox:focus#lower_chBox{

background-color:red;
}</string>
     </property>
     <property name="text">
      <string>Lower-Letter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="symbol_chBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>290</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Symbol</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="number_chBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>300</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="upper_chBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>240</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Upper-Letter</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>370</y>
       <width>591</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>200</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>100000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#password_label{
border-radius:20px;
background-color: #D1D1D1;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_pushButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#clear_pushButton{
border-radius:20px;
background-color: #610C63;
color: rgb(255, 255, 255);
}
QPushButton:hover#clear_pushButton{
background-color:#16213E;
}</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copy_pushButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>620</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#copy_pushButton{
border-radius:20px;
background-color: #610C63;
color: rgb(255, 255, 255);
}
QPushButton:hover#copy_pushButton{
background-color:#16213E;}</string>
     </property>
     <property name="text">
      <string>Copy To Clipboard</string>
     </property>
    </widget>
    <widget class="QSlider" name="length_slider">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QSlider::handle:horizontal {
border:1px solid ;
border-radius:3px;
background-color:#1B2430


}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>52</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="slider_value_label">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>140</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordQuality_label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>450</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nexa Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>626</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>C:/Users/Lenovo/Downloads/copy (3).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>545</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>C:/Users/Lenovo/Downloads/reload.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
